core_types/src/eav/query.rs

```diff
@@ -20,6 +21,7 @@ impl<'a> Default for EaviQuery<'a> {
             Default::default(),
             Default::default(),
             IndexFilter::LatestByAttribute,
+            None,
```

Comment:
I was hoping EaviQuery would stay general, and focused on only E, A, V, and I. If we start tacking on extra special-case filters, I'm afraid it will just grow and grow. Is there any reason why the tombstone filter can't be implemented as part of a simple AttributeFilter, i.e. use the `attribute` param?

Reply:
This was a space decision. Because RemoveLink has to add a RemoveLink for every other link that has this same LinkData, it will mean a tombstone will have to be set for all the link data as well. This allows a runtime tombstone and saves a lot more space, I think we might need to develop a small DSL for the EAVI imo